plumgrid_playbooks/inventory/group_vars/all.yml

```diff
@@ -22,6 +22,9 @@
 plumgrid_repo: http://<LCM-IP>:81
 plumgrid_component: halfdome-500
 
+# PLUMgrid configuration directory path
+plumgrid_data_dir: "/var/lib/libvirt/filesystems/plumgrid-data"
```

Comment:
using this path in every role couple of times, why not just declare it here, if someone want to change the data directory, can change it here easily instead of going to change everywhere